src/domains/citizen-auth-common/.terraform.lock.hcl

```diff
@@ -2,58 +2,49 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/azuread" {
-  version     = "2.33.0"
-  constraints = "<= 2.33.0"
+  version     = "2.53.0"
```

Comment:
I think some architecture is missing, run `terraform providers lock -platform=windows_amd64 -platform=darwin_amd64 -platform=darwin_arm64 -platform=linux_amd64` to check for it